delta/plugins/elasticsearch/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/elasticsearch/ElasticSearchViews.scala

```diff
@@ -136,6 +136,23 @@ final class ElasticSearchViews private (
     } yield res
   }.span("createElasticSearchView")
 
+  /**
+    * Creates a new ElasticSearchView without checking whether the provided project is deprecated or not.
+    *
+    * @param iri
+    *   id of the view to be created
+    * @param project
+    *   project in which the view has to be created
+    * @param value
+    *   configuration of the view to be created
+    */
+  def unsafeCreate(
```

Comment:
We have a similar method for deprecation called `internalDeprecate` so we could align those ?

Reply:
Updated